### PR TITLE
fix: older systems crash when playing media files

### PR DESCRIPTION
**Pull request by deermichel**
#### Description of Change

Closes https://github.com/electron/electron/issues/31762.

Regression caused by https://github.com/electron/electron/pull/31492. On Windows 7 & 8 / macOS 10.11/10.12, `system_media_controls_` is `nullptr` and calling methods on it crashes the process. This PR wraps problematic statements in an if-clause like upstream Chromium does.

I manually checked that MediaMetadata still works on Windows 10 and above.

cc @codebytere @lyswhut @miniak 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes / manually validated
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Fixed crash when playing media files on Windows 7/8 or macOS 10.11/10.12.

### CODE REVIEW

File: patches/chromium/fix_media_key_usage_with_globalshortcuts.patch
Comment (MarshallOfSound) [inline patches/chromium/fix_media_key_usage_with_globalshortcuts.patch:83]: This comment doesn't need to be moved

File: patches/chromium/fix_media_key_usage_with_globalshortcuts.patch
Comment (MarshallOfSound) [inline patches/chromium/fix_media_key_usage_with_globalshortcuts.patch:100]: I think you've broken line endings here or something? This line is marked as changed when nothing has changed, can you clean up this diff so that only changed lines are committed?

**Comment by gaodeng** [comment]
We've received similar reports from macOS El Capitan users that the entire app crashes as soon as the audio is played. Will this PR also solve the problem on macOS?

File: patches/chromium/fix_media_key_usage_with_globalshortcuts.patch
Comment (deermichel) [reply patches/chromium/fix_media_key_usage_with_globalshortcuts.patch:99]: So I tried a couple of things now, this is the cleanest version I get with `e patches chromium`. Some lines get shuffled because with the if-clause, git interprets changes a little bit different.

Comment (deermichel) [reply patches/chromium/fix_media_key_usage_with_globalshortcuts.patch:99]: If I try to remove the + or line end at end of the file, I end up with a corrupt patch.

**Review by zcbenz — APPROVED** [review]

**Comment by indutny-signal** [comment]
We'd really love to help this move forward. Please let me know if I can help with getting this merged in any way!

Thanks for the fix! ❤️

**Comment by zcbenz** [comment]
This PR requires review from @electron/wg-upgrades.

But just a notice that this organization won't have much activity this month, so even if it is merged we might have to wait until next month before it is released.

**Comment by gaodeng** [comment]
any news?

**Comment by bruceauyeung** [comment]
lately we  encountered crashes when playing video files, looking forward for this  !

**Comment by bruceauyeung** [comment]
Is there any workround to avoid crash when play media files before this PR getting merged ?
Thanks !
@zcbenz  @MarshallOfSound

**Comment by deermichel** [comment]
@bruceauyeung downgrading to 15.3.0 / 14.2.0 / 13.6.0 works

**Review by VerteDinde — APPROVED** [review]

**Comment by lyswhut** [comment]
13.x seems to be still updating,
Can it be merged into 13.x?

**Pull request merged**
